docs/make.jl

```diff
@@ -16,7 +16,8 @@ makedocs(
     format = Documenter.HTML(
         canonical = "https://juliadata.github.io/DataFrames.jl/stable/",
         assets = ["assets/favicon.ico"],
-        edit_link = "main"
+        edit_link = "main",
+        size_threshold_ignore = ["man/basics.md", "lib/functions.md"],
```

Comment:
is this a new option?

Reply:
Yes, without this option, `makedocs` will throw an error. https://github.com/JuliaDocs/Documenter.jl/pull/2252

I think `man/basics.md` is a bit long page, and it would be nice to split this page. But I would like to keep this PR small, so I added `"man/basics.md"` to `size_threshold_ignore`.